recipes/plugins.rb

```diff
@@ -40,6 +40,9 @@
 log 'Safe Restart Jenkins' do
   message 'Safe Restart Jenkins'
   only_if { ::File.exist?(reload_file) }
+end
+
+notify_group 'Safe Restart Jenkins Notify' do
   notifies :execute, 'jenkins_command[safe-restart]', :immediately
 end
```

Comment:
I think you want to remove the `log` resource and move the `only_if` to the `notify_group` resource.